components/admin/funnel/funnel-chart-svg.tsx

```
@@ -0,0 +1,265 @@
+
+			for (let b = ACTIVITY_BANDS.length - 1; b >= 0; b--) {
+				const band = ACTIVITY_BANDS[b];
+				const bandFraction = bandCounts[b] / total;
+				const outer = 1 - cumulativeFraction;
+				cumulativeFraction += bandFraction;
+				const inner = 1 - cumulativeFraction;
+				fractions.push({ band, outer, inner: Math.max(inner, 0) });
+			}
+
+			for (const { band, outer, inner } of fractions) {
+				if (outer <= inner) continue;
```

Biome (2.4.4)

[error] Block statements are preferred in this position.

(lint/style/useBlockStatements)